Rust/src/common/cluster.rs

```diff
@@ -39,7 +39,7 @@ pub trait Cluster<Q, T: ?Sized> {
     fn representatives(&self) -> Vec<(Q, f32)>;
     // a function to distill all representatives into a single representative under a given
     // distance function over the base type of the points
-    fn primary_representative(&self, distance: fn(&T, &T) -> f64) -> Q;
+    fn primary_representative(&self, _distance: fn(&T, &T) -> f64) -> Q;
```

Comment:
The primary_representative  is needed when we move to multi-centroid clustering (CURE, as in the Java version). But this change makes sense.

Reply:
Was guessing many of the unused variable / function warnings were due to not yet implemented features, so was not planning to fix any others (to avoid interfering with that). I can undo this one if you want.

Reply:
The change is fine as is -- should change it when the feature is updated.